nservicebus/azure/data-distribution_content_[5.2,6).partial.md

```diff
@@ -0,0 +1,7 @@
+snippet: UniqueQueuePerEndpointInstance
```

Comment:
@SzymonPobiega I'm afraid that won't currently work, limitation of the DocsEngine, correct @SimonCropp ?